### PR TITLE
Update generate-percentage-classes-at-breakpoints mixin with correct breakpoint suffix

### LINKED ISSUE #102: Update generate-percentage-classes-at-breakpoints to generate correct breakpoint suffix
Currently the mixin `generate-percentage-classes-at-breakpoints` generates classes like `.u-lap-one-half`, which is inconsistent with the rest of the framework. The mixin should be updated to append the breakpoint to the end eg `.u-one-half-from-lap`

### PR DESCRIPTION
**Pull request by AlecRaeside**
Closes #102

### CODE REVIEW

File: core/mixins/_generate-percentage-classes-at-breakpoints.scss [hunk 1]
```diff
@@ -52,11 +52,11 @@ $mixin-denominator-words:     (whole half third quarter fifth sixth seventh
...
 
   @each $breakpoint in $breakpoint-names {
-
-    @include respond-to($breakpoint, $limit: if($breakpoint ==
-        'palm', 'max', 'min')) {
+    $limit: if($breakpoint == 'palm', 'max', 'min');
+    @include respond-to($breakpoint, $limit) {
       @if $breakpoint {
-        $breakpoint: "#{$breakpoint}-"
+        $suffix: if($limit =='min', '-from', '-up-to');
```
Comment (chris-pearce) [inline core/mixins/_generate-percentage-classes-at-breakpoints.scss:58]: I thought we weren't catering for `max-width` mq's, see: `-up-to`?

Comment (AlecRaeside) [inline core/mixins/_generate-percentage-classes-at-breakpoints.scss:58]: not arbitrary max-width mq's but, we need this to support `palm` (which by default is a max-width mq)

Comment (chris-pearce) [inline core/mixins/_generate-percentage-classes-at-breakpoints.scss:58]: Ah ok.

File: core/mixins/_generate-percentage-classes-at-breakpoints.scss [hunk 2]
```diff
@@ -52,11 +52,11 @@ $mixin-denominator-words:     (whole half third quarter fifth sixth seventh
     $css-property);
```
Comment (chris-pearce) [inline core/mixins/_generate-percentage-classes-at-breakpoints.scss:52]: Can you make sure these comments are updated: https://github.com/AlecRaeside/scally/blob/fix-widths-mixin/core/mixins/_generate-percentage-classes-at-breakpoints.scss#L6-L21?

Comment (AlecRaeside) [inline core/mixins/_generate-percentage-classes-at-breakpoints.scss:52]: https://github.com/AlecRaeside/scally/commit/c39fb421ed599cb453e5a7dfac4c093c780fea49

**Comment by chris-pearce** [comment]
LGTM

**Comment by smgcode** [comment]
LGTM

**Pull request merged**